src/pages/LoadUnload/components/MapLegend/MapLegend.tsx

```diff
@@ -1,31 +1,76 @@
-import { Column, Row } from '@amsterdam/asc-ui'
+import { useEffect, useState } from 'react'
+
+import { Checkbox, Column, CompactThemeProvider, Label, Row } from '@amsterdam/asc-ui'
+import { useMapInstance } from '@amsterdam/react-maps'
+import { useTranslation } from 'react-i18next'
 import { MapLegend } from 'shared/components/MapLegend'
 import { MapLegendLoadUnloadSpaces } from 'shared/components/MapLegendLoadUnloadSpaces'
-import { loadUnloadSpaces } from 'shared/map/mapLayers'
+import styled from 'styled-components'
 
-import { useLoadUnloadMapContext } from '../../contexts/MapContext'
+import useLoadUnloadMapContext, { MapLayerId, layerFeatureProps } from '../../contexts/MapContext'
+import { BOLLARDS_ZOOM_LEVEL } from '../MapLayers/BollardsLayer'
 
 import { LoadUnloadMapLegendRoadSectionsLoadUnload } from './RoadSectionsLoadUnload'
 
+const StyledCheckbox = styled(Checkbox)`
+  & > span {
+    background-color: ${props =>
+      props.id && props.checked ? layerFeatureProps[props.id as MapLayerId].color : 'none'};
+  }
+`
+
 export const LoadUnloadMapLegend = () => {
   const { activeMapLayers, updateActiveMapLayers } = useLoadUnloadMapContext()
 
+  const { t } = useTranslation()
+
+  const mapInstance = useMapInstance()
+  const [bollardsAreDisabled, setBollardsAreDisabled] = useState(true)
+  useEffect(() => {
+    mapInstance.on('zoomend', () => {
+      setBollardsAreDisabled(mapInstance.getZoom() < BOLLARDS_ZOOM_LEVEL)
```

Comment:
SanityCheck: Wouldn't `!=` be better than `<` here? 
`mapInstance.getZoom() < BOLLARDS_ZOOM_LEVEL`

Reply:
No, the bollards layer disabled on levels below BOLLARDS_ZOOM_LEVEL (14?).